docs/semgrep-ci/configuration-reference.md

```diff
@@ -19,23 +19,29 @@ While environment variables are the preferred way to configure Semgrep CI, pass
 SEMGREP_RULES="p/security-audit p/secrets"
 ```
 
-## Suppressing blocking findings or errors
+## Configuring blocking findings or errors
 
-Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep exits with a non-zero exit code (exit code 1) when it reports a blocking finding or there is an issue in running Semgrep (exit code 2 and above). For more information about specific Semgrep exit codes see [CLI reference](../../cli-reference/#exit-codes).
+Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep can be configured to suppress errors (this is the default behavior) or surface all errors depending on the needs of your teams and projects. Configure this behavior by the `--suppress-errors` or `--no-suppress-errors` flags.
 
-However, Semgrep in CI can run both in fail open and fail closed states. Configure the behavior of Semgrep in CI to suppress blocking findings or internal Semgrep errors blocking your pipeline by using the following options in your YAML configuration file: 
+By default, Semgrep blocks your pipeline when it reports a blocking finding (exit code `1`). Blocking findings are defined in [Rule Board](https://semgrep.dev/orgs/-/board) of Semgrep App or encompass **all** Semgrep findings if you do **not** use Semgrep App with Semgrep in CI. Also, the default behavior is such that when Semgrep encounters errors (exit code `2` and above) your pipeline is **not** blocked. When the default behavior (equivalent to passing `--suppress-errors`) is configured, Semgrep in CI sends a crash report to a crash-reporting server and lets CI job continue. This report is anonymous and contains information that is usually sent to Semgrep App.
 
-- `semgrep ci` - Semgrep in CI **fails** on blocking findings, CI **fails** on internal errors.
-- `semgrep ci || [ $? != 1 ]` - Semgrep in CI **fails** on blocking findings, CI **passes** on internal errors.
+Configure and change the setup of blocking findings or errors in your CI pipeline by using the following options in your YAML configuration file:
+
+- `semgrep ci` - This is the default state. Semgrep in CI **fails** on blocking findings, CI **passes** on internal errors. Semgrep sends an anonymous crash report to a crash-reporting server, and then exits with exit code `0`. Optional: Define it explicitly by using `--suppress-errors` flag.
```

Comment:
@s-santillan do you think I should rather use: `run: semgrep ci` and `run: semgrep ci --no-suppress-errors` to be more specific?